### PR TITLE
feat(calling): show ellipsis for long names (AR-1148)

### PR DESCRIPTION
**Pull request by ohassine**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1148" title="AR-1148" target="_blank">StoryAR-1148</a>  View all call participants during the call (grid)
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

long names are covering the screen

### Solutions

trim long names and show ellipsis

Needs releases with:

- [ ] GitHub link to other pull request

#### How to Test

Update your name to a longer one and try to make a call

### Attachments (Optional)

Screenshot_20220627-164253

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-714/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3737556)(3737556)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3737556)(3737556)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/ui/calling/ParticipantTile.kt [hunk 1]
```diff
@@ -115,7 +117,8 @@ fun ParticipantTile(
                         bottom.linkTo(parent.bottom)
                         start.linkTo(parent.start)
                         end.linkTo((parent.end))
-                    },
+                    }
+                    .widthIn(max = dimensions().onGoingCallTileUsernameMaxWidth),
```
Comment (vitorhugods) [inline app/src/main/kotlin/com/wire/android/ui/calling/ParticipantTile.kt:121]: Is this `widthIn` necessary for it?

Can't we instead put the Text to fill the parent only?

I'm not sure if hardcoding these dimensions will resize well on all screen sizes.

**Review by vitorhugods — APPROVED** [review]
✨🚀

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-714/4/) **failed**.

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-714/5/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3745344)(3745344)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3745344)(3745344)-dev-release.apk)

**Pull request merged**